From 4fba51c82d855426ad73f8215d0ed7bfdfa4fae2 Mon Sep 17 00:00:00 2001
From: Mart Raudsepp <leio@gentoo.org>
Date: Sat, 15 Dec 2018 20:50:34 +0200
Subject: [PATCH] build: Fix error message with -Denable-vala without vala
 found

Fixes
ERROR:  Unknown variable "vala_options"
to
ERROR:  Problem encountered: you need vala >= 0.14.1 installed to use vala plugins
by correcting a typo.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index c0679d25..5647376f 100644
--- a/meson.build
+++ b/meson.build
@@ -230,7 +230,7 @@ if vala_option != 'no'
   endif
 
   if not have_vala
-    if vala_options == 'yes' or introspection_option == 'yes'
+    if vala_option == 'yes' or introspection_option == 'yes'
       error(str)
     endif
     message(str)
-- 
2.17.0

